plugins/modules/redfish_powerstate.py

```diff
@@ -77,6 +84,28 @@
        ca_path: "/path/to/ca_cert.pem"
        reset_type: "ForceOff"
        resource_id: "System.Embedded.1"
+
+- name: Perform AC Power Cycle
+  dellemc.openmanage.redfish_powerstate:
```

Comment:
Fix the indentation in the example task, it seems off. This applies to the below added new tasks too.

Reply:
Updated